### PR TITLE
Update Winapp3.ini

### PR DESCRIPTION
**Pull request by novaceb881**
Log can be found here:

https://github.com/MoscaDotTo/Winapp2/issues/419

### CODE REVIEW

File: Winapp3/Winapp3.ini [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-; DO NOT use this unless you know what you are doing and you know how to back up and restore your system.
-; This file is completely unpredictable and may even nuke your system. Use with extreme caution!
-; You have been warned. We are not responsible for damage to anyone's system.
-;
-; The first step would be to rename Winapp3.ini to Winapp2.ini so it is usable with all cleaners.
-; If you don't know how to do this, then you probably shouldn't be using this file.
-;
+; # of entries: 208
+; 
+; Winapp2.ini is fully licensed under the CC-BY-SA-4.0 license agreement. Please refer to our license agreement before using Winapp2: https://github.com/MoscaDotTo/Winapp2/blob/master/License.md
```
Comment (ROCKNROLLKID) [inline Winapp3/Winapp3.ini]: Winapp3.ini

File: Winapp3/Winapp3.ini [hunk 2]
```diff
@@ -629,10 +629,11 @@ FileKey13=%SystemDrive%\GOG Games\*|*.log;*.tmp|RECURSE
 FileKey14=%SystemDrive%\GOG Games\*\logs|*.*|RECURSE
 FileKey15=%SystemDrive%\GOG Games\*\_Redist|*.*|RECURSE
 
-[Junk File Extensions *]
+[Junk File Extensions Superfixed*]
```
Comment (ROCKNROLLKID) [inline Winapp3/Winapp3.ini]: Why?

File: Winapp3/Winapp3.ini [hunk 3]
```diff
@@ -659,85 +660,114 @@ ExcludeKey22=FILE|%SystemDrive%\Users\All Users\Microsoft\Network\Downloader\|ed
...
+ExcludeKey34=FILE|%WinDir%\System32\Msdtc\|MSDTC.LOG
+ExcludeKey35=FILE|%WinDir%\System32\Msdtc\Trace\|dtctrace.log
+ExcludeKey36=FILE|%WinDir%\System32\sru\|SRU.chk
+ExcludeKey37=FILE|%WinDir%\System32\sysprep\Panther\|setupact.log
+ExcludeKey38=FILE|%WinDir%\System32\sysprep\Panther\|setuperr.log
+ExcludeKey39=FILE|%WinDir%\System32\sysprep\Panther\IE\|setupact.log
+ExcludeKey40=FILE|%WinDir%\System32\sysprep\Panther\IE\|setuperr.log
+ExcludeKey41=PATH|%AppData%\steelseries-engine-3-client\|*.log
+ExcludeKey42=PATH|%CommonAppData%\Microsoft\Provisioning\|Microsoft-Desktop-Provisioning-Sequence.dat.LOG*
+ExcludeKey43=PATH|%CommonAppData%\Microsoft\Search\Data\Applications\Windows\|*.chk
```
Comment (ROCKNROLLKID) [inline Winapp3/Winapp3.ini]: Good to know ExcludeKeys ignore semicolons. Will remember this for the future.

File: Winapp3/Winapp3.ini (hunk 2 above)
Comment (novaceb881) [reply Winapp3/Winapp3.ini]: Sorry I have logged in now, I wanted to change it back to the original name.
I realy forgot to rename it back.

Sorry

Comment (ROCKNROLLKID) [reply Winapp3/Winapp3.ini]: That's fine. I figured it was a mistake.

File: Winapp3/Winapp3.ini (hunk 3 above)
Comment (novaceb881) [reply Winapp3/Winapp3.ini]: Would be great, if there would be an page, where we document the limitations of CCleaner.
However, I don't know the behavoir of newer versions.

**Comment by APMichael** [comment]
> Okay I found out following:
> ExcludeKey doesn't support " ; " shown here: ...

I'm sorry, but you're wrong here. ExcludeKeys have been supporting the semicolon/separator for years. You can also read it in the official documentation: https://www.ccleaner.com/docs/ccleaner/advanced-usage/ccleaner-ini-files/how-to-exclude-items-from-ccleaners-cleaning (Example 5).

I downloaded and checked CC 5.43. All ExcludeKeys work as expected. We also updated the [Junk File Extensions *] entry a few weeks ago, and the contributor confirmed that the ExcludeKeys work: https://github.com/MoscaDotTo/Winapp2/pull/397.


But there's one thing you should keep in mind:
If you specify different files using the separator ";", they always need to start with a wildcard. Without the wildcard, only the first pattern matches. (For example: ExcludeKey1=PATH|%WinDir%\|\*setup.log;\*install.log)

**Pull request merged**